HelloWorld/main.cpp

```diff
@@ -0,0 +1,28 @@
+#include <iostream>  
+using namespace std;  
+
+//Source: https://www.javatpoint.com/prime-number-program-in-cpp
+
+int isPrime(int n)
+{
+int i, m=0;
+m=n/2;  
+for(i=2; i <= m; i++)  
+{  
+	if(n % i == 0)
+			return 0;
+}  
+}
+
+int main()  
+{  
+  int n = 0;
+  cout << "Enter the Number to check Prime: ";  
+  cin >> n;  
+  int result = isPrime(n);
+  if(result)
+	  cout << "Number is Prime."<<endl;  
+  else
+    cout<<"Number is not Prime."<<endl;  
+return 0;  
```

Comment:
Probably already considered this, but: If there are ever plans to run this binary as part of a batch operation, it would be valuable to optionally take an argument, and only prompt if A) nothing is provided, and/or B) a flag is given as part of execution for interactive mode. If you decide to go this route, you'd also want to return something meaningful from the program itself, and not just 0.